config/provision/config/deploy/production.rb

```diff
@@ -0,0 +1,71 @@
+ip = '104.236.117.113'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.